v2/lints/etsi/lint_qcstatem_qcpds_lang_case.go

```diff
@@ -54,28 +46,23 @@ func isOnlyLowerCaseLetters(s string) bool {
 }
 
 func (l *qcStatemQcPdsLangCase) Execute(c *x509.Certificate) *lint.LintResult {
-	errString := ""
-	wrnString := ""
-	ext := util.GetExtFromCert(c, util.QcStateOid)
-	s := util.ParseQcStatem(ext.Value, *l.getStatementOid())
-	errString += s.GetErrorInfo()
-	if len(errString) == 0 {
-		pds := s.(util.EtsiQcPds)
-		for i, loc := range pds.PdsLocations {
-			if !isOnlyLowerCaseLetters(loc.Language) {
-				util.AppendToStringSemicolonDelim(&wrnString, fmt.Sprintf("PDS location %d has a language code containing invalid letters", i))
-			}
+	warnString := util.ErrorStringBuilder{Delimiter: "; "}
 
-		}
+	if len(c.QCStatements.ParsedStatements.PDSLocations) != 1 {
+		return &lint.LintResult{Status: lint.Error, Details: "invalid number of PdsLocations objects"}
```

Comment:
Same remark re: uniqueness; AFAICT, ETSI does not actually specify that there should be only one instance.

If I'm botching the ASN.1, I'm definitely happy to address! I'll reach out to Arno to see about raising it as a formal question.